release_notes/48-release-notes.adoc

```diff
@@ -42,7 +42,33 @@ Policy::
 
 This release adds improvements related to the following components and concepts:
 
+//ROX-20769
+[id="central-db-postgresql_15_{context}"]
+=== Central DB uses PostgreSQL 15
 
+Central DB component is using PostgreSQL 15 now, and {product-title-short} 4.8
+supports this version for external databases as well. A new installation with
+an internal database will use this version by default now. When upgrading an
+existing cluster to {product-title-short} 4.8, Central DB will do an in-place
+upgrade of the data collected so far.
+
+[NOTE]
+====
+To make the database upgrade process smooth, we recommend:
+
+* Take a backup before upgrading to {product-title-short} 4.8
```

Comment:
```suggestion
* Back up the database before upgrading to {product-title-short} 4.8
```
(Or if you want them to make a backup of everything, "Back up the system before upgrading....")